modules/lightmapper_cpu/lightmapper_cpu.cpp

```diff
@@ -725,7 +734,11 @@ void LightmapperCPU::_compute_direct_light(uint32_t p_idx, void *r_lightmap) {
 			soft_shadowing_disk_size = light.size / dist;
 
 			if (light.type == LIGHT_TYPE_OMNI) {
-				attenuation = powf(1.0 - dist / light.range, light.attenuation);
+				if (GLOBAL_GET("rendering/quality/shading/use_physical_light_attenuation")) {
+					attenuation = _get_omni_attenuation(dist, 1.0f / light.range, light.attenuation);
+				} else {
+					attenuation = powf(1.0 - dist / light.range, light.attenuation);
+				}
 			} else /* (light.type == LIGHT_TYPE_SPOT) */ {
```

Comment:
I suspect we don't want to be calling `GLOBAL_GET` with a string in a tight loop... :thinking: